toolchain/sem_ir/constant.h

```diff
@@ -11,6 +11,18 @@
 
 namespace Carbon::SemIR {
 
+// Information about a symbolic constant value. These are indexed by
+// `ConstantId`s for which `is_symbolic` is true.
+struct SymbolicConstant {
+  // The constant instruction that defines the value of this symbolic constant.
+  InstId inst_id;
+  // The enclosing generic.
```

Comment:
Is it worth explaining that `generic_id` and `index` can be invalid, and what that means?

Reply:
Done.